angular.json

```diff
@@ -339,15 +375,21 @@
         "lint": {
           "builder": "@angular-devkit/build-angular:tslint",
           "options": {
-            "tsConfig": ["libs/api/books/tsconfig.lib.json"],
-            "exclude": ["**/node_modules/**", "!libs/api/books/**/*"]
+            "tsConfig": [
+              "libs/api/books/tsconfig.lib.json"
+            ],
+            "exclude": [
+              "**/node_modules/**",
+              "!libs/api/books/**/*"
+            ]
           }
         }
       }
     }
   },
   "cli": {
-    "defaultCollection": "@nrwl/angular"
+    "defaultCollection": "@nrwl/angular",
+    "analytics": false
```

Comment:
What's the reasoning for adding this?

Reply:
When I ran 'npm run e2e' it failed with "chromeDriver doesn't support your current version" So I downloaded a compatible chromeDriver version. (https://chromedriver.storage.googleapis.com/index.html?path=85.0.4183.87/)